ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Java12/tasks/main.yml

```diff
@@ -28,20 +28,10 @@
   when: (java12_installed.stat.exists == false)
   tags: Java12
 
-- name: Test if Java12 symlink is already created
-  win_stat:
-    path: 'C:\openjdk\jdk-12'
-  register: java12_symlink
-  tags: Java12
-
-- name: Get Java 12 Folder name to create symlink
-  win_shell: powershell ([Environment]::SetEnvironmentVariable("Java12_Folder", (Get-ChildItem -Path 'C:\Program Files\Java\jdk-12*' | Select-String -Pattern 'C*'), "Machine"))
-  when: (java12_symlink.stat.exists == false)
-  tags: Java12
-
-- name: Create symlink to Java 12
-  win_shell: New-Item -Path 'C:\openjdk\jdk-12' -ItemType SymbolicLink -Value ([Environment]::GetEnvironmentVariable("Java12_Folder", "Machine"))
+- name: Create symlink to directory without spaces if not already there
+  win_shell: for /D %a in ("C:\Program Files\Java\jdk-12*") do IF NOT EXIST "C:\openjdk\jdk-12" MKLINK /D "C:\openjdk\jdk-12" "%a"
   args:
-    executable: powershell
-  when: (java12_symlink.stat.exists == false)
-  tags: Java12
+    executable: cmd.exe
+    creates: 'C:\openjdk\jdk-12'
+  tags: Java11
```

Comment:
correct tag?

Reply:
Nope - fixed. Looks like I'd fixed that on my filesystem but not pushed it back.